content/en/docs/concepts/storage/volumes.md

```diff
@@ -1179,22 +1150,23 @@ The following StorageClass parameters from the built-in vsphereVolume plugin are
 * `objectspacereservation`
 * `iopslimit`
 
-Existing volumes created using these parameters will be migrated to the vSphere CSI driver, but new volumes created by the vSphere CSI driver will not be honoring these parameters.
+Existing volumes created using these parameters will be migrated to the vSphere CSI driver,
+but new volumes created by the vSphere CSI driver will not be honoring these parameters.
 {{< /note >}}
 
-#### CSI Migration Complete
-{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+#### vSphere CSI migration complete {#vsphere-csi-migration-complete}
 
-To turn off the vsphereVolume plugin from being loaded by controller manager and kubelet, you need to set this feature flag to true. This requires `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver being installed on all worker nodes.
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
+To turn off the `vsphereVolume` plugin from being loaded by the controller manager and the kubelet, you need to set this feature flag to `true`. You must install a `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver on all worker nodes.
 
-## Using subPath
+## Using subPath {#using-subpath}
 
-Sometimes, it is useful to share one volume for multiple uses in a single Pod. The `volumeMounts.subPath`
+Sometimes, it is useful to share one volume for multiple uses in a single pod. The `volumeMounts.subPath`
 property can be used to specify a sub-path inside the referenced volume instead of its root.
 
-Here is an example of a Pod with a LAMP stack (Linux Apache Mysql PHP) using a single, shared volume.
-The HTML contents are mapped to its `html` folder, and the databases will be stored in its `mysql` folder:
+Here is an example of a `Pod` with a LAMP stack (Linux Apache Mysql PHP) using a single, shared volume.
+The HTML contents are mapped to its `html` folder and the databases are stored in its `mysql` folder:
 
 ```yaml
 apiVersion: v1
```

Comment:
aside: we ought to add a comment that this Pod is an example of using `subPath`, and isn't meant to indicate recommended practise.

Reply:
OK. Thanks.